Add Bounty Context to Treasury Page (#5307)
w3f/polkadot-wiki: docs/learn/learn-polkadot-opengov-treasury.md

[here](./learn-guides-treasury#creating-a-treasury-proposal).

:::

## Bounties

:::info Bounty Guides

See the [bounty guides](./learn-guides-bounties.md) to learn about how to
[create a bounty](./learn-guides-bounties.md#submit-a-bounty-proposal),
[assign a curator](./learn-guides-bounties.md#assign-a-curator-to-a-bounty) to a bounty, as well as
[creating and awarding child bounties](./learn-guides-bounties.md#create-and-award-child-bounties)
and [claiming child bounties rewards](./learn-guides-bounties.md#claim-a-child-bounty-reward).

:::

### Parent Bounties

Getting treasury funding through OpenGov, depending on which treasury track you submit your

> referendum, can be a long and uncertain process. This is not always a suitable option, for example, for events'
> organizers needing to pay costs upfront or close to the event's date. Bounties solve this problem by

Reviewer: ```suggestion
referendum, can be a long and uncertain process. This is not always a suitable option, for example, for event
organizers who need to pay costs upfront or close to the event's date. Bounties solve this problem by
```

> allowing access to treasury funds once to fund multiple events later on through

Reviewer: ```suggestion
procuring access to treasury funds in a single shot and use them to fund multiple events later on through
```

upfront deposit to take the position. The deposit is calculated by multiplying the curator fee by
{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMultiplier" defaultValue={500000} filter="permillToPercent"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMultiplier" defaultValue={500000} filter="permillToPercent"/> :kusama }}%,
and it can range between a minimum of
{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMin" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMin" defaultValue={3333333330} filter="humanReadable"/> :kusama }}
and a maximum of
{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMax" defaultValue={2000000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMax" defaultValue={166666666500} filter="humanReadable"/> :kusama }}.
This deposit can be used to punish curators if they act maliciously. However, if they are successful in
managing the bounty to completion, they will receive their deposit back and

> part of the bounty funding as a payment for their job.

Reviewer: ```suggestion
part of the bounty funding as a payment for their efforts.
```

> Curators are expected to have a well-balanced track record in addressing the issues that the bounty wants to

Reviewer: ```suggestion
Curators are expected to have a decent track record in addressing the issues that the bounty wants to
```

> solve. At the very least, they should be very knowledgeable on the topics covered by the bounty and have proven project
> management skills or experience. These recommendations help ensure an effective use of the bounty mechanism.

Reviewer: ```suggestion
solve. They should be very knowledgeable on the topics covered by the bounty and have proven project
management skills or experience. These recommendations help ensure an effective use of the bounty mechanism.
```

A Bounty is a reward for a specified body of work - or specified set of objectives - that needs to be

> executed for a predefined treasury amount to be paid out. The responsibility of assigning a payout

Reviewer: ```suggestion
executed for a predefined treasury amount that is designated to be paid out. The responsibility of assigning a payout
```

This track can be used for treasury spending requests up to
{{ polkadot: 100 DOT :polkadot }}{{ kusama:  8.25 KSM  :kusama }} per referendum. Each referendum
requires a decision deposit of {{ polkadot: 1 DOT :polkadot }}{{ kusama:  0.033 KSM  :kusama }}.

## Bounties

:::info Bounty Guides

See the [bounty guides](./learn-guides-bounties.md) to learn about how to
[create a bounty](./learn-guides-bounties.md#submit-a-bounty-proposal),
[assign a curator](./learn-guides-bounties.md#assign-a-curator-to-a-bounty) to a bounty, as well as
[creating and awarding child bounties](./learn-guides-bounties.md#create-and-award-child-bounties)
and [claiming child bounties rewards](./learn-guides-bounties.md#claim-a-child-bounty-reward).

:::

### Parent Bounties

Getting treasury funding through OpenGov, depending on which treasury track you submit your

> referendum, can be a long and uncertain process. This can be unideal, for example, for events'

Reviewer 2: ```suggestion
referendum, can be a long and uncertain process. This is not always a suitable option, for example, for events'
```

organizers needing to pay costs upfront or close to the event's date. Bounties solve this problem by
allowing access to treasury funds once to fund multiple events later on through

> [child bounties](#child-bounties) (that is why bounties are also called _parent_ bounties).

Reviewer 2: ```suggestion
[child bounties](#child-bounties). This is why bounties are also called _parent_ bounties.
```

> Parent bounty proposals aim to access treasury funds once to fund multiple smaller proposals later

Reviewer 2: ```suggestion
Parent bounty proposals aim to reserve a portion of treasury funds once that will be used later on.
```

> on without creating OpenGov referenda for each one of them. Bounties are managed by curators, where

Reviewer 2: ```suggestion
They save proponents the time needed to create and obtain approval for several OpenGov referenda. Bounties are managed by curators, where
```

> usually a curator is a [multi-signature account](./learn-account-multisig.md). Bounties can access a

Reviewer 2: ```suggestion
the curator is usually a [multi-signature account](./learn-account-multisig.md). Bounties can access a
```

large amount of funds, so managing those funds with a multisig is a good practice to enhance
security. Essentially, curators are multisig addresses with agency over a portion of the treasury to
promote events, fix a bug or vulnerability, develop a strategy, or monitor a set of tasks related to
a specific topic: all for the benefit of the
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} ecosystem.

A proposer can [submit a bounty proposal](./learn-guides-bounties.md#submit-a-bounty-proposal) to
OpenGov,
[with a curator to be defined later](./learn-guides-bounties.md#assign-a-curator-to-a-bounty), whose
background and expertise is such that they can determine when the task is complete.

> When submitting the value of the bounty, the proposer includes a fee paid to curators willing to

Reviewer 2: ```suggestion
When submitting the value of the bounty, the proposer can specify a fee that will be paid to curators willing to
```

> invest their time and expertise in the task: this amount is included in the total value of the

Reviewer 2: ```suggestion
invest their time and expertise in the task: this amount will be included in the total value of the
```

> bounty. In this sense, the curator's fee can be defined as subtracting the value paid to the bounty

Reviewer 2: ```suggestion
bounty. In this sense, the curator's fee can be defined as the difference between the amounts paid to child bounty
```

> rewardee from the total value of the bounty.

Reviewer 2: ```suggestion
awardees and the total value of the bounty.
```

> Curators are selected through OpenGov referendum after the bounty proposal passes and need to add an

Reviewer 2: ```suggestion
Curators are selected through OpenGov referendum after the bounty proposal passes; and they need to pay an
```

upfront deposit to take the position. The deposit is calculated by multiplying the curator fee by
{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMultiplier" defaultValue={500000} filter="permillToPercent"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMultiplier" defaultValue={500000} filter="permillToPercent"/> :kusama }}%,
and it can range between a minimum of
{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMin" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMin" defaultValue={3333333330} filter="humanReadable"/> :kusama }}
and a maximum of
{{ polkadot: <RPC network="polkadot" path="consts.bounties.curatorDepositMax" defaultValue={2000000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.curatorDepositMax" defaultValue={166666666500} filter="humanReadable"/> :kusama }}.

> This deposit can be used to punish them if they act maliciously. However, if they are successful in

Reviewer 2: ```suggestion
This deposit can be used to punish curators if they act maliciously. However, if they are successful in
```

> their task of getting someone to complete the bounty work, they will receive their deposit back and

Reviewer 2: ```suggestion
managing the bounty to completion, they will receive their deposit back and
```

> part of the bounty reward as a fee payment for their job.

Reviewer 2: ```suggestion
part of the bounty funding as a payment for their job.
```

> Curators are expected to have a well-balanced track record related to the issues the bounty tries to

Reviewer 2: ```suggestion
Curators are expected to have a well-balanced track record in addressing the issues that the bounty wants to
```

> resolve: they should be at least knowledgeable on the topics the bounty touches and show project

Reviewer 2: ```suggestion
solve. At the very least, they should be very knowledgeable on the topics covered by the bounty and have proven project
```

> management skills or experience. These recommendations ensure an effective use of the mechanism. A

Reviewer 2: ```suggestion
management skills or experience. These recommendations help ensure an effective use of the bounty mechanism.
```

> Bounty is a reward for a specified body of work - or specified set of objectives - that needs to be

Reviewer 2: ```suggestion
A Bounty is a reward for a specified body of work - or specified set of objectives - that needs to be
```

executed for a predefined treasury amount to be paid out. The responsibility of assigning a payout
address once the specified set of objectives is completed is delegated to the curator.

The bounty has a predetermined duration of
{{ polkadot: <RPC network="polkadot" path="consts.bounties.bountyUpdatePeriod" defaultValue={1296000} filter="blocksToDays"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.bounties.bountyUpdatePeriod" defaultValue={1296000} filter="blocksToDays"/> :kusama }}

> days with the possibility of being extended by the curator. Aiming to maintain flexibility on the

Reviewer 2: ```suggestion
days, with possible extension(s) to be requested by the curator. To maintain flexibility during the
```

> tasks’ curation, the curator will be able to create child bounties for more granularity and

Reviewer 2: ```suggestion
tasks’ curation, the curator will also be able to create child bounties for more granularity in the allocation of funds and 
```

> allocation in the next iteration of the mechanism.

Reviewer 2: ```suggestion
as part of a nested iteration of the bounty mechanism.
```

### Child Bounties

> Child bounties are spawned from [parent bounties](#parent-bounties). Child bounties access funds

Reviewer 2: ```suggestion
Child bounties are spawned from [parent bounties](#parent-bounties). Child bounties are used to access funds
```
